### PR TITLE
Add binary_search_recursive Algorithm

### PR DESCRIPTION
**Pull request by VivekWesley**
Add binary_search Algorithm using recursion

### CODE REVIEW

File: Task 1/move_common_elements_lefSide_algorithm/move_common_leftside.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Algorithm:
```
Comment (chandrikadeb7) [inline Task 1/move_common_elements_lefSide_algorithm/move_common_leftside.py:1]: Add question.md file for this

Comment (VivekWesley) [reply Task 1/move_common_elements_lefSide_algorithm/move_common_leftside.py:1]: okay

Comment (VivekWesley) [reply Task 1/move_common_elements_lefSide_algorithm/move_common_leftside.py:1]: Added question.md file

**Comment by VivekWesley** [comment]
Please review my PR

**Review by nimishbongale — COMMENTED** [review]
Good job @VivekWesley!

As a final review, you may want to check the `README.md` file again. Task 3 doesn't require you to have a question.md, but you could have the sample input & output as a part of a multiline comment at the beginning of the file. Task 1 however, requires the `question.md` file. Just a small change, I know you can do it! 

Let's get this merged! 🚀

File: Task 3/binary_search_recursive_approach/question.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# [Q] Serach element using binary search using recursion
```
Comment (nimishbongale) [inline Task 3/binary_search_recursive_approach/question.md:1]: Task 3 does not require a markdown file. You can add the input output as a part of a multiline comment in the beginning, or move this program to Task 1.

Comment (VivekWesley) [reply Task 3/binary_search_recursive_approach/question.md:1]: okay

**Comment by nimishbongale** [comment]
I believe this is now just a duplicate of #181 ? I'll go ahead and close this PR then.

**Comment by VivekWesley** [comment]
Yes, Thanks.

**Pull request closed without merging**